### ISSUE TITLE
Asset building/handling not covered in documentation

### ISSUE THREAD

**Issue by loganb** [tongueroo/ufo#37]
Hello, 

Neat tool! I'm trying it out now and after reading the docs, I have an unanswered question that I both seek an answer to and simultaneously recommend you cover in your FAQ. Specifically, I'm trying to figure out how you handle asset building when using ufo. 

In my current Dockerfile, I set RAILS_ENV using a build arg. A docker build command for me looks something like: `docker  build  --build-arg RAILS_ENV=production --tag analytics-web:production-2018-05-12-XXXX .` The assets then get built with the configuration of that environment, and RAILS_ENV gets set for the image by the `Dockerfile` rather than being set at runtime. 

This presents two questions:
- Since UFO doesn't seem to have a way to build environment-specific images, how do you recommend assets be built? Do you build them at deploy-time? Or do you build them into the image itself? 
- Is there a way to control how ufo generates the tags to, e.g., tag staging/development/production builds distinctly? 

Thank you for sharing your hard work automating ECS deploys. I appreciate any answers you can share and also recommend incorporating them into the FAQ.

**Comment by tongueroo** [comment]
@loganb RE: assets
Right now, what I’ve usually done is build assets into the image. Guess as a hack you could run asset package multiple times with the different RAILS_ENV. 

RE: more control over tagging. 
Can’t be done. Wondering a good approach. 

RE: FAQ
Will add once figure out this issue. 

Going to think about this issue more and get back soon. 

PS. Also just fixed my email notifications. So hopefully will see issues little faster.

**Comment by tongueroo** [comment]
@loganb RE: assets and build options in the `docker build` command
docker build options can be now be specified with the `UFO_DOCKER_BUILD_OPTIONS` environment variable. Added to [FAQ](http://ufoships.com/docs/faq/)

RE: more control over tagging.
Best can come up with right now is to re-tag the docker image after building it. Added to [FAQ](http://ufoships.com/docs/faq/): 

```
$ ufo docker build
$ ufo docker name
tongueroo/hi:ufo-2018-05-19T11-41-06-6714713
$ docker tag $(ufo docker name) hi:mytag
$ docker images | grep hi
hi                                                     mytag                              5b01e38bd060        3 minutes ago       955MB
tongueroo/hi                                           ufo-2018-05-19T11-41-06-6714713    5b01e38bd060        3 minutes ago       955MB
$ docker push hi:mytag
```

The changes have been released in version v3.5.4 https://github.com/tongueroo/ufo/blob/master/CHANGELOG.md

Closing out issue. Thanks for the feedback.
